templates/internet-of-things/digital-signage.html

```diff
@@ -101,7 +101,7 @@ <h3 class='p-heading--four'><a class='p-link--external' href='https://www.bright
           <h4 class='p-heading-icon__title'>Case study</h4>
         </div>
       </div>
-      <h3 class='p-heading--four'><a class='p-link--external' href='/blog/cinergy-makes-significant-digital-signage-savings-using-ubuntu-core' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'digital signage - right', 'eventLabel' : 'Case study - Find out how Cinergy saves on digital Signage', 'eventValue' : '1' });'>Find out how Cinergy saves on digital Signage</a></h3>
+      <h3 class='p-heading--four'><a href='/engage/secure-digital-signage-screenly' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'digital signage - right', 'eventLabel' : 'Case study - Screenly innovates IoT fleet management for secure digital signage', 'eventValue' : '1' });'>Screenly innovates IoT fleet management for secure digital signage</a></h3>
```

Comment:
Optionally I know it's not your doing, but this can look clearer if we indent the `<a ..` into a new line?